tracing-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/tracing/autoconfig/SpanProperties.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2010-2020. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.tracing.autoconfig;
+
+import org.axonframework.extensions.tracing.MessageTag;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+/**
+ * Class containing properties defining the {@link io.opentracing.Span} build by this extension.
+ *
+ * @author Corrado Musumeci
+ * @author Steven van Beelen
+ * @since 4.4
+ */
+@ConfigurationProperties(prefix = "axon.extension.tracing.span")
+public class SpanProperties {
+
+    /**
+     * A list of {@link MessageTag} instances defining the tags to be added to a {@link io.opentracing.Span} when
+     * dealing with a {@link org.axonframework.commandhandling.CommandMessage}.
+     * <p>
+     * Defaults to the tags {@link MessageTag#MESSAGE_ID}, {@link MessageTag#MESSAGE_TYPE}, {@link
+     * MessageTag#PAYLOAD_TYPE} and {@link MessageTag#MESSAGE_NAME}.
+     */
+    private List<MessageTag> commandTags =
+            asList(MessageTag.MESSAGE_ID, MessageTag.MESSAGE_TYPE, MessageTag.PAYLOAD_TYPE, MessageTag.MESSAGE_NAME);
+    /**
+     * A list of {@link MessageTag} instances defining the tags to be added to a {@link io.opentracing.Span} when
+     * dealing with a {@link org.axonframework.eventhandling.EventMessage}.
+     * <p>
+     * Defaults to the tags {@link MessageTag#MESSAGE_ID}, {@link MessageTag#AGGREGATE_ID}, {@link
+     * MessageTag#MESSAGE_TYPE} and {@link MessageTag#PAYLOAD_TYPE}.
+     */
+    private List<MessageTag> eventTags =
```

Comment:
There is any reason for the `event` to not have the `messageName`? Is it the same as the `payloadType` in this case?

Reply:
To be honest, I am not entirely sure. It wasn't there through the `SpanUtils`, so we figured let's keep it the same.

If I could place a hunch, it is because only the `CommandMessage` and `QueryMessage` have an additional `commandName`/`queryName` field, whereas `EventMessage` instances don't have that.
Hence, the `messageName` of an `EventMessage` will *always* default to the `payloadType`, making this field slightly duplicated. 

So, I feel it would be okay as is, although I understand the frowns. :-)